data_model/fees_executor/src/parameters.rs

```diff
@@ -0,0 +1,21 @@
+use alloc::{format, string::String, vec::Vec};
+
+use iroha_executor_data_model::parameter::Parameter as ExecutorParameter;
+use iroha_schema::IntoSchema;
+use iroha_data_model::prelude::*;
+use serde::{Deserialize, Serialize};
+
+#[derive(PartialEq, Eq, ExecutorParameter, Deserialize, Serialize, IntoSchema)]
+pub struct FeesOptions {
+    pub receiver: AccountId,
+    pub asset: AssetDefinitionId,
+}
+
+impl Default for FeesOptions {
```

Comment:
I don't think that we can have this default. You should find a different approach IMO

Reply:
Changed so they are obtained from the environment variables.

Reply:
This envis used only when building the executor and is the uploader's responsibility. It shouldn't be affected by individual peers' configurations.

Reply:
Well I'll be damned. This is not a bad solution at all